cv_bridge/src/CMakeLists.txt

```diff
@@ -8,6 +11,10 @@ target_link_libraries(${PROJECT_NAME} ${Boost_LIBRARIES} ${OpenCV_LIBRARIES})
 
 install(TARGETS ${PROJECT_NAME} DESTINATION lib)
 
+install(FILES
+  ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}/${PROJECT_NAME}_export.h
+  DESTINATION include/${PROJECT_NAME})
+
```

Comment:
Similar to over here https://github.com/ros2/teleop_twist_joy/pull/14#issuecomment-542380900, removing this section seems to cause it not to install the header, which is a bit unexpected given the documentation

Reply:
Ah, I apologize, after rereading I found this line: 
> The resulting file should be installed with other headers in the library.